### PR TITLE
Created 'nltk' library installation guide - FIXES #96

### LINKED ISSUE #96: Create documentation with guide for how to install the 'nltk' library
Documentation is needed in the form of a guide for how to install the 'nltk' library. This is needed as this library has caused installation issues for most people who have tried to work on the project, and these issues are not intuitive to solve. Therefore a documentation guide should be created for how to install the 'nltk' library successfully so that it is not such a hassle for future users.

### PR DESCRIPTION
**Pull request by TravisHerrick7**
Fixes #96 

**What was changed?**
Nothing was "changed", I created a new .md file that has the installation steps for the nltk library for MacOS, Windows, and Linux. 

*Here, describe what part of the application you changed (e.g. login page, database, etc.). If possible, mention what specific components were added, removed, or modified.*
I created a new documentation file titled nltk-installation-guide.md that contains the guide for installing nltk.

**Why was it changed?**
This was created because the nltk library was giving everyone issues who has tried to develop on this project and there was no documentation for this very troubling issue that is not intuitive to solve. Therefore, I created documentation for future developers to look at so that they can avoid this hassle.

*Here, describe the issue that you are fixing with this code, and why your code fixes it.*
I fixed the issue of there being no guide for installing nltk on every OS, as my file has a guide for each OS to install nltk.

**How was it changed?**
Creating a new .md file with the guide to download nltk on each OS.

*Here, get into detail about what files you modified, and talk about the most important lines in regards to fixing the issue.*
I created a new file nltk-installation-guide.md and the important lines are dependent on which OS you use as that section will contain the guide to installing it on your OS.

**Screenshots that show the changes (if applicable):**
N/A

### CODE REVIEW

**Review by kate-holdener — CHANGES_REQUESTED** [review]
@TravisHerrick7 Looks like you accidentally deleted a lot of important files. Please undo the delete. @loganwyas FYI

File: nltk-installation-guide.txt
Comment (kate-holdener) [inline nltk-installation-guide.txt:1]: I suggest making this a .md file (instead of .txt)

**Review by kate-holdener — DISMISSED** [review]

**Comment by TravisHerrick7** [comment]
> @TravisHerrick7 Looks like you accidentally deleted a lot of important files. Please undo the delete. @loganwyas FYI

@kate-holdener Sounds good. I will look into this.

**Comment by loganwyas** [comment]
@TravisHerrick7 How close is this to being completed?

**Comment by TravisHerrick7** [comment]
> @TravisHerrick7 How close is this to being completed?

Fairly close. I haven't completed it yet as I was out of town this weekend and have been working today. I will get this ready to merge tonight.

**Review by loganwyas — APPROVED** [review]

**Pull request merged**